Health/src/main/java/com/sciencesquad/health/health/Nutrition/RealmNutritionModel.java

```diff
@@ -0,0 +1,57 @@
+package com.sciencesquad.health.health.Nutrition;
+
+
+import android.support.annotation.NonNull;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+
+/**
+ * Created by danielmiller on 2/11/16.
+ *
+ * Realm Model for Nutrition database.
+ * Things that are relevant to store for the Nutrition Module go here.
+ */
+public class RealmNutritionModel extends RealmObject {
```

Comment:
Yes! Simple:
```
RealmNutritionModel model = getDataFromSomewhere();
RealmNutritionModel update = RealmNutritionModel.copy(model).calorieIntake(38574).build();
putDataSomewhere(update);
```

**Copying is always preferred over mutability.** ^remember that, kids.